Hippo.Web/Hippo.Web.csproj

```diff
@@ -26,7 +26,6 @@
     <PackageReference Include="Serilog.Enrichers.ClientInfo" Version="1.1.4" />
     <PackageReference Include="Serilog.Exceptions" Version="8.0.0" />
     <PackageReference Include="Serilog.Sinks.Elasticsearch" Version="8.4.1" />
-    <PackageReference Include="YamlDotNet" Version="13.1.0" />

```

Comment:
Did you mean to remove it from here?

Reply:
Yes. It's now in Hippo.Core. The versions conflicted, so it seemed cleaner to have it specified in one location rather than trying to keep them in sync.